docs/core/pipes/date-time.pipe.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: Date Time Pipe
+Added: v1.0.0
```

Comment:
Looking at others md files this should be version of actual ADF which is 6.2.0

Reply:
added correct version